components/src/core/components/Input/Select/SelectInput.vue

```diff
@@ -124,6 +124,10 @@ export default defineComponent({
       type: Boolean,
       default: true,
     },
+    forcedDirection: {
```

Comment:
Previous implementation refers the direction as `dropdownPosition` , even tho semantically correct word maybe is 'direction', its better to match the name with 'position'

eg: 'forceDropdownPosition', 'fixDropdownPosition' etc